google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsTracer.java

```diff
@@ -268,6 +273,15 @@ public void grpcMessageSent() {
     grpcMessageSentDelay.set(attemptTimer.elapsed(TimeUnit.NANOSECONDS));
   }
 
+  /*
+  In GrpcClientCalls we set the timeout in ApiContext on CallOptions. The timeout in ApiContext will be the attempt
```

Comment:
This comment needs to be updated:

This is called by BigtableTracerCallables that sets operation timeout from user settings.